doc/architecture/reference_points_coordinate_systems.adoc

```diff
@@ -0,0 +1,16 @@
+= Reference points and coordinate systems
+
+With Open Simulation Interface, an object's position coordinates refer to the center of the object's bounding box, independent of the coordinate system. This reference point is not identical to the origin of any coordinate systems.
+
+OSI uses three coordinate systems:
+
+World coordinates::
+Coordinate system for all quantities which are part of ground truth.
+
+Sensor coordinates::
+Coordinate system for all quantities which are part of sensor data.
+
+Object coordinates::
+Coordinate system for local object coordinates, for example, vehicle coordinates.
+
```

Comment:
For all coordinate systems: Would it make sense to say something about the origin? e.g.
World coordinates: inertial x/y/z positions
Sensor coordinates: mounting position of the sensor / virtual mounting position
Object coordinates: For vehicles it is currently middle rare axle, but there were discussions about maybe changing it to middle bounding box

Reply:
**Output CCB 09.06.2021:**
1. Describe OSI's Coordinate systems in a generic manner.
2. Describe how to define the position of the coordinate systems in OSI (for example wrt reference points, which may be the center of the bounding box).
3. Example picture to illustrate the above mentioned points (a picture is already available needs to be updated).